### ISSUE TITLE
Simplify control argument names

### ISSUE THREAD

**Issue by jeffkeller87** [RSGInc/RSGHB#76]
Control argument names are outdated, occasionally inconsistent, and confusing (particularly the "normal" and "fixed" nomenclature). I've set the HBControl function up to easily catch deprecated arguments and direct users to new usages/names. I don't think this is a pressing issue at all, but wanted to get my thoughts down before I lose them.

Below are my suggestions: currentName (proposedName)

- gVarNamesNormal (???)
- gVarNamesFixed (???)
- gDIST (dist)
- FC (???)
- svN (???)
- gNCREP (???)
- gNEREP (???)
- gNSKIP (skip)
- gINFOSKIP (trace)
- priorVariance (pv)
- degreesOfFreedom (df)
- constraintsNorm (???)
- rho/rhoF (???)
- targetAcceptanceNormal (???)
- targetAcceptanceFixed (???)
- gFULLCV (???)
- gStoreDraws (keep.draws)
- gSeed (seed)
- gMINCOEF (???)
- gMAXCOEF (???)
- nodiagnostics (diagnostics; invert to avoid double negative)

**Comment by jeffkeller87** [comment]
Makes sense to me to combine FC+gVarNamesFixed and svN+gVarNamesNormal into two numeric vectors with optional names to simplify things.

E.g. c(beta_price = 0, beta_time = -0.5)

This is related to issue #75.

**Comment by jeffdumont** [comment]
How about?


   - gVarNamesNormal (par.random.names)
   - svN (par.random.sv)
   - gDIST (par.random.dist)
   - constraintsNorm (par.random.constraints)
   - gVarNamesFixed (par.fixed.names)
   - FC (par.fixed.sv)
   - gNCREP (iter.burnin)
   - gNEREP (iter.model)
   - gNSKIP (iter.skip)
   - gINFOSKIP (iter.trace)
   - priorVariance (pv)
   - degreesOfFreedom (df)
   - rho/rhoF (rho.random/rho.fixed)
   - targetAcceptanceNormal (target.ar.random)
   - targetAcceptanceFixed (target.ar.fixed)
   - gFULLCV (full.covariance)
   - gStoreDraws (keep.draws)
   - gSeed (seed)
   - gMINCOEF (can be deprecated and specified by the user in the
   likelihood function)
   - gMAXCOEF (can be deprecated and specified by the user in the
   likelihood function)
   - nodiagnostics (diagnostics; invert to avoid double negative)

**Comment by jeffkeller87** [comment]
I like these. What do you think about combining par.random.sv/.names and par.fixed.sv/.names down to par.random and par.fixed (optionally named)?

**Comment by jeffdumont** [comment]
That is okay with me.
